src/cljs/chronograph_web/pages/organization/views.cljs

```diff
@@ -1,10 +1,95 @@
+      {:placeholder "Name"
+       :value name
+       :on-change #(rf/dispatch [::org-events/create-task-form-update
+                                 :name
+                                 (.-value (.-currentTarget %))])}]
+     [components/text-input :description :tasks/description
+      {:placeholder "Description"
+       :value description
+       :on-change #(rf/dispatch [::org-events/create-task-form-update
+                                 :description
+                                 (.-value (.-currentTarget %))])}]
+     [:button {:type :button
+               :name :create
+               :disabled (or (= status :creating)
+                             form-invalid?)
+               :on-click (fn [] (rf/dispatch [::org-events/create-task-form-submit]))}
+      (if (= status :creating)
+        "Creating..."
+        "Create")]]))
+
+(defn update-form [{:keys [id] :as _task}]
+  (let [{:keys [form-params status]} @(rf/subscribe [::subs/update-task-form id])
+        form-invalid? (not (form-valid? form-params))
+        {:keys [name description]} form-params]
+    [:form
+     [components/text-input :name :tasks/name
+      {:placeholder "Name"
+       :value name
+       :on-change #(rf/dispatch [::org-events/update-task-form-update
+                                 id
+                                 :name
+                                 (.-value (.-currentTarget %))])}]
+     [components/text-input :description :tasks/description
+      {:placeholder "Name"
+       :value description
+       :on-change #(rf/dispatch [::org-events/update-task-form-update
+                                 id
+                                 :description
+                                 (.-value (.-currentTarget %))])}]
+     [:button {:type :button
+               :name :save
+               :disabled (or (= status :saving)
+                             form-invalid?)
+               :on-click (fn [] (rf/dispatch [::org-events/update-task-form-submit id]))}
+      (if (= status :saving)
+        "Saving..."
+        "Save")]
+     [:button {:type :button
+               :name :save
+               :disabled (or (= status :saving)
+                             form-invalid?)
+               :on-click (fn [] (rf/dispatch [::org-events/cancel-update-task-form id]))}
+      "Cancel"]]))
+
+(defn task-list-element [{:keys [id name description is-updating] :as task}]
+  [:li {:key id}
+   [:div
+    (if (true? is-updating)
+      [update-form task]
+      [:p [:b name] " - " description])
+    [:button {:on-click #(rf/dispatch [::org-events/archive-task id])}
+     "Archive"]
+    [:button {:on-click #(rf/dispatch [::org-events/show-update-form task])}
+     "Update"]]])
+
+(defn task-list [tasks]
+  [:div
+   [:h3 "Tasks"]
+   [:ul
+    (not-empty (map task-list-element tasks))]])
+
 (defn organization-page [{:keys [slug]}]
   (rf/dispatch [::org-events/fetch-organization slug])
+  (rf/dispatch [::org-events/fetch-tasks slug])
```

Comment:
As explained earlier, these two events could be collapsed into a single `page-mounted` event.

Reply:
As discussed on call — let's leave the implementation as-is.